frontend/src/component/admin/users/EditUser/EditUser.tsx

```diff
@@ -77,9 +77,8 @@ const EditUser = () => {
         <FormTemplate
             loading={loading}
             title="Edit user"
-            description="In order to get access to Unleash needs to have an Unleash root role as Admin, Editor or Viewer.
-            You can also add the user to projects as member or owner in the specific projects."
-            documentationLink="https://docs.getunleash.io/reference/rbac#standard-roles"
+            description="In order to manage user access to Unleash, the user needs to be assigned a root role, such as Viewer, Editor, or Admin."
```

Comment:
Same suggestion as above, but maybe we want to tailor the text more towards editing here? 🤔 

```suggestion
            description="In order for a user to get access to Unleash, they need to be assigned a root role, such as Viewer, Editor, or Admin."
```